src/RefractAPI.cc

```diff
@@ -142,23 +142,25 @@ namespace drafter
     refract::IElement* ParameterValuesToRefract(
         const NodeInfo<snowcrash::Parameter>& parameter, ConversionContext& context)
     {
-        refract::ArrayElement* element = CollectionToRefract<refract::ArrayElement>(
-            MAKE_NODE_INFO(parameter, values), context, LiteralToRefract<std::string>, SerializeKey::Enum);
+        refract::EnumElement* element = new refract::EnumElement;
+
+        refract::ArrayElement* enumerations = CollectionToRefract<refract::ArrayElement>(
+            MAKE_NODE_INFO(parameter, values), context, LiteralToRefract<std::string>);
 
         // Add sample value
         if (!parameter.node->exampleValue.empty()) {
-            refract::ArrayElement* samples = new refract::ArrayElement;
-            samples->push_back(
-                CreateArrayElement(LiteralToRefract<std::string>(MAKE_NODE_INFO(parameter, exampleValue), context)));
-            element->attributes[SerializeKey::Samples] = samples;
+            element->set(LiteralToRefract<std::string>(MAKE_NODE_INFO(parameter, exampleValue), context));
         }
 
         // Add default value
         if (!parameter.node->defaultValue.empty()) {
             element->attributes[SerializeKey::Default]
-                = CreateArrayElement(LiteralToRefract<std::string>(MAKE_NODE_INFO(parameter, defaultValue), context));
+                = LiteralToRefract<std::string>(MAKE_NODE_INFO(parameter, defaultValue), context);
         }
 
+        // Add enumeraitons
```

Comment:
This comment has a typo (`enumeraitons`).

I'm not entirely sure that this comment adds any value as it is obvious this code does what the comment says. Generally commenting why something is done is valuable, but what something done doesn't seem useful.